src/context/ProductReducerFunction.js

```diff
@@ -0,0 +1,61 @@
+// import {
+// 	addItemToCart,
+// 	addItemToWishlist,
+// 	getWishlist,
+// 	removeItemFromWishlsit,
+// } from "../Api/apicalls";
+const productReducerFunction = (state, action) => {
+	switch (action.type) {
+		case "PRICE_RANGE_FILTER":
+			return {
+				...state,
+				dataFilter: { ...state.dataFilter, maxPrice: Number(action.payload) },
+			};
+		case "SORT_BY_PRICE":
+			if (action.payload === "LOW_TO_HIGH") {
+				return {
+					...state,
+					dataFilter: { ...state.dataFilter, sortByPrice: "LOW_TO_HIGH" },
+				};
+			} else if (action.payload === "HIGH_TO_LOW") {
+				return {
+					...state,
+					dataFilter: { ...state.dataFilter, sortByPrice: "HIGH_TO_LOW" },
+				};
+			} else {
+				return state;
```

Comment:
suggestion: I think you can directly assign action.payload to the sortByPrice instead of checking action.payload using if-else. Like 
case "SORT_BY_PRICE":
				return {
					...state,
					dataFilter: { ...state.dataFilter, sortByPrice: "LOW_TO_HIGH" },
				};

Reply:
yes. Implemented in new commit. Not sure why I used if else in first place.